contracts/contracts/FleekERC721.sol

```diff
@@ -57,7 +57,6 @@ contract FleekERC721 is Initializable, ERC721Upgradeable, FleekAccessControl {
         string ENS; // ENS ID
         uint256 currentBuild; // The current build number (Increments by one with each change, starts at zero)
         mapping(uint256 => Build) builds; // Mapping to build details for each build number
-        string[] accessPoints; // List of app AccessPoint
```

Comment:
Created this card so we make sure we create the subgraph updates for it: https://app.shortcut.com/fleekxyz/story/458/create-events-and-handlers-for-ap.